### DISCUSSION TITLE
Neo TV [BO]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#375] [Show and tell]
### Channel ID

NeoTV.bo

### Stream URL

https://vdo.streams.ovh:3702/live/neotvlive.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Stream does not work: geo-blocked?

**Comment by freearhey** [comment]
https://vdo.streams.ovh:3702/live/neotvlive.m3u8 (offline)
<img width="360" alt="image" src="https://user-images.githubusercontent.com/7253922/232254423-2642213d-572a-464f-8769-e663f31452f1.png">
